Gemfile.lock

```diff
@@ -192,4 +195,4 @@ RUBY VERSION
    ruby 2.6.1p33
 
 BUNDLED WITH
-   2.0.2
+   2.1.4
```

Comment:
just FYI this is usually a no-no if you have automated deployment tools because it won't be able to adjust automatically. I'll deploy this manually so it's not a big deal, but for future reference gotta be careful about this.